src/Interpreters/getClusterName.cpp

```diff
@@ -46,7 +46,7 @@ std::string getClusterName(const IAST & node)
 }
 
 
-String getClusterNameAndMakeLiteral(ASTPtr & node)
+std::string getClusterNameAndMakeLiteral(ASTPtr & node)
```

Comment:
Because `getClusterName.h` includes only `<string>` - 3fd760a39919bd3a9ae4ce65b6f6ff07211d4630, it works because this header included after inclusion of `base/common/types.h` everywhere